naive-serverless-map-reduce/pi.js

```diff
@@ -0,0 +1,21 @@
+exports.computePiMapper = body => {
+  let inside = 0;
+
+  // repeat points times:
+  for (let i = 0; i < body.points; i++) {
```

Comment:
Do you need to use a for-loop here. Couldn't you just do a map and then reduce down to the number of inside points? Even just a map instead of a for loop would make it better.